examples/load-data-and-cgm-stats.r

```diff
@@ -0,0 +1,120 @@
+####################################################################################
+# description: Example for loading Tidepool data and calculating cgm stats in R
+# version: 0.0.1
+# created: 2018-07-17
+# author: Jason Meno
+# license: BSD-2-Clause
+####################################################################################
+
+# Set location of data
+dataPath = "../example-data/"
+
+# load csv data
+data = read.csv(paste0(dataPath,
+                       "example-from-j-jellyfish.csv"),
+                       stringsAsFactors = FALSE)
+
+# View the first 5 rows of data
+head(data)
+
+# Get a list of all data columns
+colnames(data)
+
+# Get the unique data types
+unique(data$type)
+
+# get just the cgm data
+cgm = data[data$type=="cbg",]
+
+# look at the first 5 rows of data
+head(cgm)
+
+# rename "value" field to "mmol_L"
+colnames(cgm)[colnames(cgm)=="value"] = "mmol_L"
+
+# convert mmol/L to mg/dL and create a new field
+cgm$mg_dL = as.integer(cgm$mmol_L * 18.01559)
+
+# view the cgm mg/dL data
+head(cgm$mg_dL)
+
+# define a function that captures the Ambulatory Glucose Profile statistics
+# http://www.agpreport.org/agp/agpreports#CGM_AGP
+get_stats <- function(df){
```

Comment:
I suggest/prefer to use "=" over "<-" despite the fact "<-" is the legacy R way of doing things.

Reply:
Fixed, "<-" replaced with "="